src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -25,8 +27,10 @@ const AnonymousChannelListScreen = ({route}) => {
     channels: anonChannels,
     goToChatScreen,
     goToPostDetailScreen,
-    goToContactScreen
+    goToContactScreen,
+    goToCommunityScreen
   } = useAnonymousChannelListScreenHook();
+  const {fetchLatestTopicPost} = useSignedChannelListScreenHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useSignedChannelListScreenHook` hook is being used to fetch the latest topic post. However, this seems to be a signed operation and might not work as expected in an anonymous context. Please ensure that this operation can be performed anonymously or consider creating a similar function in `useAnonymousChannelListScreenHook` for fetching the latest topic post.
```diff
- const {fetchLatestTopicPost} = useSignedChannelListScreenHook();
+ const {fetchLatestTopicPost} = useAnonymousChannelListScreenHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->